Dockerfile.base

```diff
@@ -38,76 +38,46 @@ RUN apt-get update -qq && apt-get install -y -qq    \
                     clang-tidy                      \
                     jq                              \
                     kmod                            \
-                    llvm                            \
                     lld                             \
+                    llvm                            \
                     llvm-dev                        \
-                    libasio-dev                     \
                     libclang-dev                    \
                     libhwloc-dev                    \
                     libjemalloc-dev                 \
                     libboost-atomic-dev             \
-                    libboost-chrono-dev             \
-                    libboost-date-time-dev          \
-                    libboost-filesystem-dev         \
-                    libboost-iostreams-dev          \
-                    libboost-program-options-dev    \
                     libboost-regex-dev              \
-                    libboost-system-dev             \
-                    libboost-thread-dev             \
                     libgoogle-perftools-dev         \
                     mpi-default-dev                 \
                     vc-dev                          \
-                    doxygen                         \
-                    python3                         \
-                    python3-pip                     \
-                    texlive                         \
-                    texlive-latex-extra             \
-                    latexmk                         \
                     libjson-perl                    \
```

Comment:
Do you know if this is used? May be possible to remove?

Reply:
Yes I'm not sure, I'll give it a try